test/jdk/java/lang/Math/WorstCaseTests.java

```diff
@@ -267,6 +288,9 @@ private static int testWorstAcos() {
             {+0x1.4182199998587p-1,     +0x1.C8A538AE83D1Fp-1},
             {+0x1.E45A1C93651ECp-1,     +0x1.520DC553F6B23p-2},
             {+0x1.F10FC61E2C78Fp-1,     +0x1.EFEEF61D39AC1p-3},
+
+            // Worst-case observed error
+            {-0x1.0068b067c6feep-1,     +0x1.0c335e2f0727p1},
```

Comment:
Similar considerations as for asin above.
The "expected" value should be `0x1.0c335e2f0726fp1`.

Reply:
For the new worst-case conditions I've pushed, if there is a "// check" comment, the Math and StrictMath return values were the same and I did not yet check which exact floating-point value should be used as the reference point.